NEWS

```diff
@@ -28,7 +28,13 @@ None
 None
 
 *** Removed
-None
+- da-dk.dis: This table has long been superseded by da-dk-octobraille.dis.
+- da-lt.ctb: This table was a step on the way to defining a Danish 8 dot
+standard. The table was referring to an old Danish Braille note-taker
+from the 80s and 90s, which is not used any more.
+- fi1.ctb and fi2.ctb: according to the Finnish Federation of the 
+Visually Impaired, These tables are obsolete (see issue #814). Use
```

Comment:
Drop the reference to the GH issue. Also I'm not sure we need to mention the Finnish Federation of the 
Visually Impaired here.